core/src/test/java/org/elasticsearch/search/query/QueryStringIT.java

```diff
@@ -161,8 +157,6 @@ public void testDocWithAllTypes() throws Exception {
         assertHits(resp.getHits(), "1");
         resp = client().prepareSearch("test").setQuery(queryStringQuery("Baz")).get();
         assertHits(resp.getHits(), "1");
-        resp = client().prepareSearch("test").setQuery(queryStringQuery("sbaz")).get();
-        assertHits(resp.getHits(), "1");
```

Comment:
Added a general ngram search rest test to analysis module, so the removal of these lines and above is covered.